### DISCUSSION TITLE
How to show both title and all parents of note that is currently open

### DISCUSSION THREAD

**Discussion by MotikaCraft-TheJoshua** [TriliumNext/Trilium#4359] [Q&A]
So basically, instead of showing only note's title/name, is it possible to also show all parents and parents of parents until last one that is currently hoisted (on image, root is parent of all parents)?

![image](https://github.com/zadam/trilium/assets/56804505/9cb91d46-75cc-4327-b2d5-36dd1fee5444)

Thank you in advance!

**Comment by zerebos** [comment]
You might be interested in a widget like [Trilium Breadcrumbs](https://github.com/rauenzi/Trilium-Breadcrumbs) that allows you to do this.

https://user-images.githubusercontent.com/6865942/261645406-e3578d3d-78d9-4c32-84d6-063023a02f5f.mp4
